Manifest.txt

```diff
@@ -1,17 +1,12 @@
 CHANGELOG.md
 CODE_OF_CONDUCT.md
-CONTRIBUTING.md
 LICENSE.txt
-MAINTAINERS.txt
 MIT.txt
 Manifest.txt
-POLICIES.md
 README.md
-UPGRADING.md
 bundler/CHANGELOG.md
 bundler/LICENSE.md
 bundler/README.md
-bundler/UPGRADING.md
```

Comment:
Is it intentional that all these files were removed from the release package? The PR description only mentioned unifying documentation, but not changing the contents of the package the we release.

Reply:
I will look later.